packages/gestalt/src/Switch.css

```diff
@@ -1,5 +1,7 @@
 :root {
   --g-switch-height: 24px;
+  --g-vr-slider-height-on: 18px;
```

Comment:
can we avoid vr if there's no conflicting name? we can do `--g-slider-height-on: 18px;` this prevents us from having to rename thigs later on

Reply:
Done, I was thinking of explicitly using VR names to make it easier to find which changes are VR later, but I suppose we can use commits for that